tests/data/testConnection/config.json

```diff
@@ -9,7 +9,6 @@
       "database": "test",
       "user": "sa",
       "#password": "yourStrong(!)Password"
-    },
-    "tables": []
```

Comment:
asi lepsi pouzivat novy dir `rowTestConnection` a nechat tenhle config jak to je

Reply:
nice catch :)